File: local/HDR__/python/shape/shape/circle.py
#circle.py
from shape.shape import Shape

class Circle(Shape):
    def __init__(self, x, y, rad):
        super().__init__(x, y)
        self.__rad = rad
        
    def area(self):
        return 3.14 * self.__rad * self.__rad
    def get_circumference(self):
        return 3.14 * 2 * self.__rad
    

File: local/HDR__/cpp/cpp/shape/circle.cpp
#include "shape.h"

Circle::Circle(int x, int y, int rad)
: Shape(x, y), rad_(rad)
{

}

double Circle::area()const{
    return 3.14 * rad_ * rad_;
}

double Circle::getCircumference() const{
    return 2 * rad_ * 3.14;
}

